packages/crayons-core/package.json

```diff
@@ -98,9 +97,12 @@
   },
   "license": "MIT",
   "dependencies": {
+    "@freshworks/crayons-icon": "file:../crayons-icon",
     "@popperjs/core": "^2.10.2",
     "@stencil/core": "2.9.0",
-    "moment-mini": "^2.24.0"
+    "moment-mini": "^2.24.0",
+    "hast-util-to-html": "^8.0.3",
```

Comment:
These imports are not used in this package. Kindly move them to the correct package where they are being used.

Reply:
closed

Reply:
This is not closed. Can you please fix it?